arbicoin/settings.py

```diff
@@ -37,6 +37,10 @@
     'django.contrib.sessions',
     'django.contrib.messages',
     'django.contrib.staticfiles',
+    'transactions',
+    'blocks',
+    'rest_framework',
```

Comment:
Try to follow the following pattern of app inclusion:
- First party
- Third Party
- local/project